### PR TITLE
deal with systemd MAC address assignment

### PR DESCRIPTION
**Pull request by ytinirt**
Signed-off-by: ZHAO Yao <zhao.yao@h3c.com>

### CODE REVIEW

File: images/base/Dockerfile [hunk 1]
```diff
@@ -121,6 +121,8 @@ COPY [ "entrypoint", "/usr/local/bin/" ]
 # NOTE: this is *only* for documentation, the entrypoint is overridden by the node image
 ENTRYPOINT [ "/usr/local/bin/entrypoint", "/sbin/init" ]
 
-# TODO(bentheelder): deal with systemd MAC address assignment
+# deal with systemd MAC address assignment
 # https://github.com/systemd/systemd/issues/3374#issuecomment-288882355
 # https://github.com/systemd/systemd/issues/3374#issuecomment-339258483
+RUN cp /usr/lib/systemd/network/99-default.link /etc/systemd/network/ \
```
Comment (aojea) [inline images/base/Dockerfile]: Is this file guarantee to be there in newer versions?

File: images/base/Dockerfile [hunk 2]
```diff
@@ -121,6 +121,8 @@ COPY [ "entrypoint", "/usr/local/bin/" ]
 # NOTE: this is *only* for documentation, the entrypoint is overridden by the node image
 ENTRYPOINT [ "/usr/local/bin/entrypoint", "/sbin/init" ]
 
-# TODO(bentheelder): deal with systemd MAC address assignment
+# deal with systemd MAC address assignment
```
Comment (aojea) [inline images/base/Dockerfile]: @BenTheElder can you give more context on this issue? I don´t remember related failures

Comment (BenTheElder) [reply images/base/Dockerfile]: it's just a known issue with some virtual devices and systemd that I noted for the future, I'm not sure that this is in fact an issue we've had an problem with yet. so far as far as I can tell it is not.

Comment (BenTheElder) [reply images/base/Dockerfile]: it might be equally valid to just remove this comment

Comment (BenTheElder) [reply images/base/Dockerfile]: we went that route, thanks again!
https://github.com/kubernetes-sigs/kind/pull/590
/close

**Pull request closed without merging**